src/actions/navigation.js

```diff
@@ -23,15 +26,14 @@ function willNavigate() {
  * @static
  */
 function navigated() {
-  return ({ dispatch }) => {
-    // We need to load all the sources again because they might have
-    // come from bfcache, so we won't get a `newSource` notification.
-    //
-    // TODO: This seems to be buggy on the debugger server side. When
-    // the page is loaded from bfcache, we still get sources from the
-    // *previous* page as well. For now, emulate the current debugger
-    // behavior by not showing sources loaded by bfcache.
-    // return dispatch(sources.loadSources());
+  return ({ dispatch, client, getState }) => {
```

Comment:
We have a slightly nicer style here:
https://github.com/arthur801031/debugger.html/blob/update_source/src/actions/pause.js#L45-L48